src/platform_impl/windows/monitor.rs

```diff
@@ -178,17 +207,37 @@ impl MonitorHandle {
     pub fn size(&self) -> PhysicalSize<u32> {
         let monitor_info = get_monitor_info(self.0).unwrap();
         PhysicalSize {
-            width: (monitor_info.rcMonitor.right - monitor_info.rcMonitor.left) as u32,
-            height: (monitor_info.rcMonitor.bottom - monitor_info.rcMonitor.top) as u32,
+            width: (monitor_info
+                .__AnonymousBase_winuser_L13571_C43
+                .rcMonitor
+                .right
+                - monitor_info
+                    .__AnonymousBase_winuser_L13571_C43
+                    .rcMonitor
+                    .left) as u32,
```

Comment:
@kennykerr You may be interested in this generated code.

Reply:
There's already an issue open for this: https://github.com/microsoft/win32metadata/issues/732